src/components/Package/Package.tsx

```diff
@@ -155,7 +159,7 @@ const Package: React.FC<PackageInterface> = ({
             <PackageTitle className="package-title">{packageName}</PackageTitle>
           </WrapperLink>
         </Grid>
-        <GridRightAligned item={true} xs={true}>
+        <GridRightAligned alignItems="center" container={true} item={true} justify="flex-end" xs={true}>
```

Comment:
Does this PR requires change this `alignItems="center" container={true} item={true} justify="flex-end"``?

Reply:
yes! it was necessary to do this changes